subprojects/android-test/test-screenshot/src/main/java/com/avito/android/test/screenshot_test/internal/ScreenshotDirectories.kt

```diff
@@ -0,0 +1,39 @@
+package com.avito.android.test.screenshot_test.internal
+
+import android.annotation.SuppressLint
+import java.io.File
+
+internal class ScreenshotDirectories(private val packageName: String) {
+
+    operator fun get(type: String): File {
+        return getSdcardDir(type)
+    }
+
+    private fun getSdcardDir(type: String): File {
+        val externalStorage = System.getenv("EXTERNAL_STORAGE")
+            ?: throw RuntimeException(
```

Comment:
IllegalStateException
Could we check this env earlier? Maybe when we create the `ScreenshotDirectories `

Reply:
can`t get what kind of problem it will solve